mne/epochs.py

```diff
@@ -662,7 +662,7 @@ def __next__(self, *args, **kwargs):
     def __hash__(self):
         if not self.preload:
             raise RuntimeError('Cannot hash epochs unless preloaded')
-        return object_hash(dict(info=self.info, data=self._data))
+        return object_hash(dict(info=hash(self.info), data=self._data))
```

Comment:
same here -- does the hash of an old/master `Epochs` object equal what it is under this PR? It should, since we aren't actually changing anything about the content of the object (just how we do some sanity checking under the hood).